lib/dh.js

```diff
@@ -100,16 +89,23 @@ function DH(prime, generator, malleable) {
   this._primeLen = prime.length;
   this._pub = void 0;
   this._priv = void 0;
-  
+  this._primeCode = void 0;
```

Comment:
This is really uncommon, could you put a comment to explain what the `void 0` achieves here?